lib/poseidon/consumer_group.rb

```diff
@@ -94,10 +94,10 @@ def self.pick(pnum, cids, id)
   # @option options [Boolean] :trail Starts reading messages from the latest partitions offsets and skips 'old' messages . Default: false
   #
   # @api public
-  def initialize(name, brokers, zookeepers, topic, options = {})
+  def initialize(name, brokers, zk, topic, options = {}, before_callback = nil, after_callback = nil)
```

Comment:
I prefer to pass the callbacks as options, i.e. `:before_rebalance` and `:after_rebalance`, instead of arguments to initialize.